docs/apps/transfer/authorizations.md

```diff
@@ -0,0 +1,47 @@
+# `TransferAuthorization`
+
+`TransferAuthorization` implements the `Authorization` interface for `ibc.applications.transfer.v1.MsgTransfer`. It allows a granter to grant a grantee the privilege to submit MsgTransfer on its behalf. Please see the [Cosmos SDK docs](https://docs.cosmos.network/v0.47/modules/authz) for more details on granting privileges via the `x/authz` module.
+
+More specifically, the granter allows the grantee to transfer funds over a specified channel that belong to the granter.
+
+For the specified channel, the granter shall be able to specify a spend limit of a specific denomination they wish to allow the grantee to be able to transfer.
```

Comment:
ditto on "shall" wording

Reply:
+ it's also required to set the spend limit right so perhaps 'must' is better here